app/policies/api/tables_policy.rb

```diff
@@ -0,0 +1,19 @@
+class Api::TablesPolicy < ApplicationPolicy
+  def create?
+    # Assuming that only admin users can create a new table
+    if user.is_a?(User) && user.admin?
+      true
+    else
+      raise Pundit::NotAuthorizedError, "You do not have permission to create a new table."
+    end
```

Comment:
The `Api::TablesPolicy` is assuming that only admin users can create a new table. Ensure that this assumption is correct and that the appropriate permissions are being checked.